test/smoke-fails/dyn_libs/Makefile.dynlibs

```diff
@@ -0,0 +1,42 @@
+#===--------- libm/Makefile -----------------------------------------------===
+#
+#                The LLVM Compiler Infrastructure
+#
+# This file is distributed under the University of Illinois Open Source
+# License. See LICENSE.TXT for details.
+#
+#===----------------------------------------------------------------------===
+#
+# Makefile:  Makefile to build static device library as archive of bc files
+#            written by Greg Rodgers
+#
+include ../../Makefile.defs
+
+LIBA      = A
+LIBA      = B
+LIBASRC   = libA
+LIBBSRC   = libB
+
+CXX     = $(AOMP)/bin/clang++
+CFLAGS  = $(AOMP_CPUTARGET) -fopenmp -fopenmp-targets=$(AOMP_GPUTARGET) \
+            -Xopenmp-target=$(AOMP_GPUTARGET) \
+            -march=$(AOMP_GPU) -O2
```

Comment:
Should this use `--offload-arch=$(AOMP_GPU)` instead of the more verbose syntax?